src/Pelagos/Bundle/AppBundle/Resources/public/js/dataset-monitoring.js

```diff
@@ -6,12 +6,17 @@ $(document).ready(function() {
     // Due to a bug in hashChange (old version, no newer compatible with current jQuery),
     // manualy detect the hash on load, and force hash change.
     // But we have to wait with opening nodes until js Tree is fully loaded.
-    $("#tree").bind("loaded.jstree", function(e, data) {
+    let tree = $("#tree");
+    tree.bind("loaded.jstree", function(e, data) {
         if (window.location.hash != "") {
             $(window).trigger("hashchange");
         }
     });
 
+    tree.on('select_node.jstree', function (e, data) {
+        data.instance.toggle_node(data.node);
```

Comment:
The ticket says to do nothing when already expanded, so may have to check the state of data.node.

Reply:
They thought it would be more work for us to close the tree on click, i'll check with the PO

Reply:
This works to detect it, if the PO really wants it to not re-close: if (data.instance.is_closed(data.node))

Reply:
POs were okay with the expand and collapse on click